lib/WorkflowMag.groovy

```diff
@@ -85,14 +85,15 @@ class WorkflowMag {
             }
         }
 
+        // Check MetaBAT2 inputs
+        if ( !params.skip_metabat2 && params.min_contig_size < 1500 ) {
+            log.warn "Specified min. contig size under minimum for MetaBAT2. MetaBAT2 will be run with 1500 (other binners not affected). You supplied: --min_contig_size ${params.min_contig_size}"
+        }
+
         // Check if at least two binners were applied in order to run DAS Tool for bin refinment
         // (needs to be adjusted in case additional binners are added)
-        if (params.refine_bins_dastool && params.skip_metabat2 ) {
-            log.error 'Both --refine_bins_dastool and --skip_metabat2 are specified! Invalid combination, bin refinement requires MetaBAT2 and MaxBin2 binning results.'
-            System.exit(1)
-        }
-        if (params.refine_bins_dastool && params.skip_maxbin2 ) {
-            log.error 'Both --refine_bins_dastool and --skip_maxbin2 are specified! Invalid combination, bin refinement requires MetaBAT2 and MaxBin2 binning results.'
+        if ( params.refine_bins_dastool && [ params.skip_metabat2, params.skip_maxbin2, params.skip_concoct ].count(false) <= 1 ) {
```

Comment:
Doesnt `.count(false) <= 1` mean that 1 or less of the binners are run?
I had in mind to check whether 2 or more binners (> 1) are not skipped (false), following line 93 `Check if at least two binners were applied in order to run DAS Tool for bin refinment`:
```suggestion
        if ( params.refine_bins_dastool && [ params.skip_metabat2, params.skip_maxbin2, params.skip_concoct ].count(false) > 1 ) {
```

Reply:
Ah, yeah sorry. I flipped the use case here, as I want to throw an error if it is less than 1.

But using > 1 means you don't have to adjust in the future (I just need to take the opposite to throw the error), like you suggest :+1: 

Added a comment for myself as I'm very bad at thinking about these things abstractly.

Reply:
I've tested now @d4straub! If you're OK with it can you merge.

Reply:
Haha true, I had it wrong here! I am happy with the change!

Reply:
Ok, once tests pass I merge (ignoring the template lint stuff)